index.md

```diff
@@ -960,6 +960,41 @@ Command Examples:
       LIST >3                         ; Command to list all channels with
                                       more than three users.
 
+### INVITE message
+
+         Command: INVITE
+      Parameters: <nickname> <channel>
+      Alt Params: 0
+
+The INVITE command is used to invite a user to a channel.  The parameter `<nickname>` is the nickname of the person to be invited to the target channel `<channel>`.
+
+The target channel SHOULD exist (at least one user is on it).  Otherwise, the server SHOULD reject the command with the `ERR_NOSUCHCHANNEL` numeric.
+
+Only members of the channel are allowed to invite other users.  Otherwise, the server MUST reject the command with the `ERR_NOTONCHANNEL` numeric.
+
+When the channel has [invite-only](#invite-only-channel-mode) mode set, only channel operators may issue INVITE command.  Otherwise, the server MUST reject the command with the `ERR_CHANOPRIVSNEEDED` numeric.
```

Comment:
After investigation, this isn't really accurate:

* Plexus requires OP only if the channel is -i or +p ("private"/"paranoia")
* Charybdis allows non-op to invite on +i channels if +g ("open-invite") is set.

Plus, as @Adam pointed out, we don't see why the spec should require the ircd to reject invites for not being chanop ever

So we could reword it like this:

> > Servers MAY reject the command with the `ERR_CHANOPRIVSNEEDED` numeric. For example, they may reject it when the channel has [invite-only](#invite-only-channel-mode) mode set, and the user is not a channel operator.

Or this stricter version:

> Servers MAY reject the command with the `ERR_CHANOPRIVSNEEDED` numeric. In particular, they SHOULD reject it when the channel has [invite-only](#invite-only-channel-mode) mode set, and the user is not a channel operator.

Reply:
Applied the suggestion in https://github.com/ircdocs/modern-irc/pull/80/commits/402fdafc8c26031fc935b8e36fa8e05f58e41c74

Should other paragraph be reworded? (e.g. replace MUST to SHOULD)

Reply:
I'd rather keep these MUST. All servers I tested do send these numerics, and it's useful for clients to have some guarantee they'll get a reply when the server processed their request.